pages/apim/3.x/user-guide/publisher/service-discovery/service-discovery-eureka.adoc

```diff
@@ -5,3 +5,66 @@
 :page-title: Eureka service discovery
 
 include::https://raw.githubusercontent.com/gravitee-io/gravitee-service-discovery-eureka/master/README.adoc[]
```

Comment:
I would have replaced this "include" with a direct link to the github repository. It would make the rendered page lighter.

Reply:
I agree. I changed it with a link.